src/hooks/useAccomodationQuery.ts

```diff
@@ -0,0 +1,9 @@
+import { useQuery } from "@tanstack/react-query";
+import getAccomodationInfo from "@/api/accomodation/getAccomodationInfo";
+
+export const useAccomodationQuery = (id: number) => {
+  return useQuery({
+    queryKey: ["accomodationInfo"],
+    queryFn: () => getAccomodationInfo(id)
```

Comment:
```suggestion
export const useAccomodationQuery = (id: number) => {
  return useQuery({
    queryKey: ["accomodationInfo", id],
    queryFn: () => getAccomodationInfo(id)
```
queryFn에서 사용되는 변수가 있다면 이 변수가 queryKeys에도 포함되어야 합니다

제가 queryKey에 어떤 값을 넣어야하는지 제대로 설명을 못드린 것 같아서 
https://github.com/ssi02014/react-query-tutorial#usequery 에서 쿼리키 부분 설명 참고 부탁드립니다..!
https://tanstack.com/query/v4/docs/react/guides/query-keys#if-your-query-function-depends-on-a-variable-include-it-in-your-query-key